<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Ingredient_DB</class>
 <widget class="QWidget" name="F_Ingredient_DB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>243</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>296</width>
    <height>243</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>296</width>
    <height>243</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="F_Ingredient_DB_Layout">
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QFrame" name="Ingredient_Frame">
     <property name="minimumSize">
      <size>
       <width>278</width>
       <height>225</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>278</width>
       <height>225</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="Ingredient_Layout">
      <item>
       <layout class="QHBoxLayout" name="Name_Layout">
        <item>
         <widget class="QLabel" name="Name_Label">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Название</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Name_TBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Volume_Layout">
        <item>
         <widget class="QLabel" name="Volume_Label">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Объем (мл)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Volume_TBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Price_Layout">
        <item>
         <widget class="QLabel" name="Price_Label">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Цена</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Price_TBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Alcohol_Layout">
        <item>
         <widget class="QLabel" name="Alcohol_Label">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Алкоголь (%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Alcohol_TBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Delete_Layout">
        <item>
         <widget class="QPushButton" name="Delete_Button">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Recources.qrc">
            <normaloff>:/Icons/Icons/Delete_Icon.png</normaloff>:/Icons/Icons/Delete_Icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Accept_Button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Recources.qrc">
            <normaloff>:/Icons/Icons/Accept_Icon.png</normaloff>:/Icons/Icons/Accept_Icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Recources.qrc"/>
 </resources>
 <connections/>
</ui>
